packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -204,6 +211,28 @@ const HomePage = () => {
     });
   };
 
+  const handleDeleteMediaFromModal = async id => {
+    handleClickToggleModal();
+    const overlayblockerParams = {
+      children: <div />,
+      noGradient: true,
+    };
+    strapi.lockApp(overlayblockerParams);
+
+    try {
+      await deleteMedia(id);
+
+      dispatch({
+        type: 'ON_DELETE_MEDIA_SUCCEEDED',
+        mediaId: id,
+      });
+    } catch (err) {
+      // Silent
```

Comment:
Should we display a notification here?

Reply:
The notification is already displayed with the `mediaDelete`